RELEASING.md

```diff
@@ -189,6 +189,78 @@ Central (the staging repository will be destroyed in the process). You can see
 the complete process for releasing to Maven Central on the [OSSRH
 site](http://central.sonatype.org/pages/releasing-the-deployment.html).
 
+## Announcement
+
+Once deployment is done, go to Github [release
+page](https://github.com/census-instrumentation/opencensus-java/releases), press
+`Draft a new release` to write release notes about the new release.
+
+You can use the Github [compare
+tool](https://github.com/census-instrumentation/opencensus-java/compare/)
+to view a summary of all commits since last release as a reference.
+Please pick major or important changes only.
+
+## Update release versions in documentations and build files
+
+After releasing is done, you need to update all readmes and examples to point to the
+latest version.
+
+1. Update README.md and gradle/maven build files on `master` branch:
+
+```bash
+$ git checkout -b bump-document-version master
+$ RELEASE_VERSION_FILES=(
+  README.md
+  contrib/grpc_util/README.md
+  contrib/http_util/README.md
```

Comment:
contrib/zpages must be updated as well

Reply:
Do you mean contrib/zpages/README.md? It should be included in next line.

Reply:
Clearly I need to go to an eye doctor 👍